backend/stylists/public/js/MerchantProducts.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", function(event) {
+    var approveAll = document.getElementsByClassName('approve')[0].getElementsByTagName('input')[1];
+    var rejectAll = document.getElementsByClassName('approve')[0].getElementsByTagName('input')[2];
+    var selectedIds = [];
+    approveAll.addEventListener('click', function(e) {
+        selectedIds = getSelectedProductIds();
+        if (selectedIds.length <= 0){
+            alert('Select items to approve');
```

Comment:
if (selectedIds.length == 0) then
give a confirm box "Do you want to approve/reject all listed items "
If answer is yes then call getAllProductIds
else
e.preventDefault(e);